libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4231,7 +4231,7 @@ MAV_RESULT GCS_MAVLINK::handle_command_do_set_mission_current(const mavlink_comm
 {
     AP_Mission *mission = AP::mission();
     if (mission == nullptr) {
-        return MAV_RESULT_UNSUPPORTED;
+        return MAV_RESULT_FAILED;
```

Comment:
I think this is incorrect.  This indicates the mission library is simply missing - i.e. we really don't support missions :-)

No items will be handled as part of the `set_current_cmd` on line 4238

Reply:
Arrrg. Thanks!